CHANGELOG.md

```diff
@@ -17,6 +17,9 @@ The types of changes are:
 
 ## [Unreleased](https://github.com/ethyca/fides/compare/2.20.0...main)
 
+### Fixed
+- Enable toggles in various tables now render an error toast if an error occurs [#4095](https://github.com/ethyca/fides/pull/4095)
```

Comment:
If going out in 2.20 you could put this in the changelog below instead!

Reply:
ah right!